src/formats/Molfile.cpp

```diff
@@ -335,13 +333,7 @@ template<> FormatInfo chemfiles::format_information<Molfile<DCD>>() {
 
 template<> FormatInfo chemfiles::format_information<Molfile<TRJ>>() {
     return FormatInfo("TRJ").with_extension(".trj").description(
-        "GROMACS .xtc binary format"
-    );
-}
-
-template<> FormatInfo chemfiles::format_information<Molfile<XTC>>() {
-    return FormatInfo("XTC").with_extension(".xtc").description(
-        "GROMACS .xtc binary compressed portable format"
+        "GROMACS .trj binary format"
```

Comment:
great typo spotting =). BTW, do you also plan in replacing the TRJ reader?

Reply:
Thanks. I'm only working with XTC/TRR trajectories so reading them was somewhat of a high priority for me. I probably should stop procrastinating for a while, so I wouldn't have time for this. :/ But if I have a bit more time in a while and nobody else did it until then I could give it a try.

Reply:
ok =)